src/components/Product/ProductGrid.js

```diff
@@ -0,0 +1,24 @@
+import Product from './Product';
+import styled from 'styled-components';
+
+export default function ProductGrid({ products }) {
+	return (
+		<ProductUl>
+			{products.map(product => (
+				<ProductLi key={product.id}>
+					<Product detail={product.detail} email={product.email} image={product.image} />
+				</ProductLi>
+			))}
+		</ProductUl>
+	);
+}
+
+const ProductUl = styled.ul`
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+`;
+
+const ProductLi = styled.li`
+	list-style: none;
+`;
```

Comment:
Maybe you want to reuse the list-style an another point? Think about creating a separate styled-component for the list?

Reply:
Thanks Ursula, my next  UserStory is about reorganize my styled component files. I started the project with a misunderstanding on how to work properly with styled comments. I already started this branch when I got the noticed of the misleading style component folder structure. So I would leave the List item to the next branch.